index.js

```diff
@@ -33,6 +34,17 @@ function readFile(filename, json) {
   return json ? JSON.parse(file) : file;
 }
 
+function getFiles(dir) {
```

Comment:
How about we use `globby` here to make this logic easier - and allow user to specify glob pattern matching?